test/TaskQueue_test.js

```diff
@@ -0,0 +1,99 @@
+const assert = require('assert');
+const debug = require('debug')('index:test:api_test');
+const helper = require('./helper');
+const slugid = require('slugid');
+const taskcluster = require('taskcluster-client');
+const request = require('superagent');
+const assume = require('assume');
+const libUrls = require('taskcluster-lib-urls');
+
+helper.secrets.mockSuite('TaskQueue_test.js', ['taskcluster'], function(mock, skipping) {
+  helper.withFakeQueue(mock, skipping);
+
+  test('Run task and test indexing', async function() {
```

Comment:
I think this test title comes from `services/index/test/index_test.js`.  Let's rewrite it to describe what this test is doing.